cfme/ansible_tower/explorer.py

```diff
@@ -164,6 +201,18 @@ class TowerExplorerJobTemplatesAll(CFMENavigateStep):
     prerequisite = NavigateToAttribute('appliance.server', 'AnsibleTowerExplorer')
 
     def step(self, *args, **kwargs):
-        self.view.sidebar.job_templates.tree.click_path(VersionPicker({Version.lowest():
-            'All Ansible Tower Job Templates', '5.10': 'All Ansible Tower Templates'}).pick(
-            self.view.browser.product_version))
+        self.view.sidebar.job_templates.tree.click_path('All Ansible Tower Templates')
+
+
+@navigator.register(AnsibleTowerJobTemplate, 'Details')
+class TowerExplorerJobTemplateDetails(CFMENavigateStep):
+    VIEW = TowerExplorerJobTemplateDetailsView
+    prerequisite = NavigateToAttribute('parent', 'All')
+
+    def step(self, *args, **kwargs):
+        self.prerequisite_view.toolbar.view_selector.select('List View')
+        try:
+            row = self.prerequisite_view.entities.get_entity(name=self.obj.name)
```

Comment:
Q: if the number of pages more than one then?

Reply:
`surf_pages=True`! :smiley: 
thanks, I missed it